extensions/interactions/ImageClickInput/directives/ImageClickInput.js

```diff
@@ -80,19 +80,15 @@ oppia.directive('oppiaInteractiveImageClickInput', [
             };
             $scope.loadImage();
           } else {
+            $scope.isLoadingIndicatorShown = false;
+            $scope.isTryAgainShown = false;
             // This is the case when user is in exploration editor. We don't
             // have loading indicator or try again for showing images in the
             // exploration editor. So we directly fetch the images from the
             // AssetsBackendApiService's cache.
-            AssetsBackendApiService.loadImage(
+            $scope.imageUrl = AssetsBackendApiService.getImageUrlForPreview(
               ExplorationContextService.getExplorationId(),
-              $scope.filepath)
-              .then(function(loadedImageFile) {
-                $scope.isLoadingIndicatorShown = false;
-                $scope.isTryAgainShown = false;
-                var objectUrl = URL.createObjectURL(loadedImageFile.data);
-                $scope.imageUrl = objectUrl;
-              });
+              $scope.filepath);
```

Comment:
This will fit at the previous line, no need for linebreak.

Reply:
Done!